xray/src/bin/merge_xray_quadtrees.rs

```diff
@@ -0,0 +1,23 @@
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "merge_xray_quadtrees")]
+/// Merge partial xray quadtrees. We assume that the root
+/// of each quadtree belongs to the same level of the final
+/// quadtree.
+struct CommandlineArguments {
+    /// Directory where to write the merged quadtree. Does *not*
+    /// have to be disjoint from input_directories.
+    #[structopt(parse(from_os_str), long = "output_directory")]
+    output_directory: PathBuf,
```

Comment:
You can remove the `long`, as default in our Rust code is kebap case and structopt automatically converts to it.

Reply:
Do you mean:
`#[structopt(parse(from_os_str), long)]`
or
`#[structopt(parse(from_os_str))]`?
The second option would make the argument positional.

Reply:
Oh yes, sorry for the confusion, I meant `long`

Reply:
:+1: